int16 leftRpm
int16 rightRpm
---
bool success
